.github/pytorch-probot.yml

```diff
@@ -1,6 +1,7 @@
 # The schema is from https://github.com/pytorch/pytorch/blob/main/.github/pytorch-probot.yml
 ciflow_push_tags:
 - ciflow/android
+- ciflow/android/release
```

Comment:
I think it makes more sense to not use `ciflow/android/release` as it's a hack.  The ciflow tag is for PR.  If you want to keep the current implementation, you could name it something else I guess to avoid confusion, i.e.  `push/to/this/tag/to/release/android`